benchmarks/viscoelastic_sheared_torsion/viscoelastic_sheared_torsion.prm

```diff
@@ -137,19 +138,21 @@ subsection Material model
     set Densities                   =  1
 
     set Elastic shear moduli        = 1.e2
+
+    # Even though we do not use a fixed elastic time step, this parameters needs to be set equal to the "maximum time step"
+    # as it is used to compute time step 0.
```

Comment:
```suggestion
    # as it is used to compute the elastic time step during time step 0.
```

Reply:
Also, can you split line 142 into two lines? That makes sure the lines fit fully on a small screen (we try to do that with all code). Same for line 149 :)